packages/rs-dapi-client/src/executor.rs

```diff
@@ -55,12 +79,48 @@ pub struct ExecutionResponse<R> {
     pub address: Address,
 }
 
-impl<R> ExecutionResponse<R> {
+impl<R, I> IntoInner<I> for ExecutionResponse<R>
+where
+    R: Into<I>,
+{
     /// Unwrap the response
-    pub fn into_inner(self) -> R {
-        self.inner
+    fn into_inner(self) -> I {
+        self.inner.into()
+    }
+}
+
+impl<T: VersionedGrpcResponse> VersionedGrpcResponse for ExecutionResponse<T> {
```

Comment:
In DAPI client we shouldn't know about metadata and proofs

Reply:
We don't use it actually. Let's remove it.

Reply:
removed